src/content/SectionPopup/SectionPopup.tsx

```diff
@@ -30,6 +30,8 @@ const SectionPopup = ({
         <div className="SectionPopupTitle">{selectedSection?.code}</div>
         <hr />
         <div className="SectionPopupDetails">{selectedSection?.name}</div>
+        <div className="SectionPopupInstructor">Instructor: </div>
+        <div>{selectedSection?.instructor || "No Instructor Information"}</div>
```

Comment:
Can we change the message to "unavailable" to keep it consistent with the grades information

Reply:
Done.